rules/windows/defense_evasion_persistence_account_tokenfilterpolicy.toml

```diff
@@ -0,0 +1,72 @@
+[metadata]
+creation_date = "2022/11/01"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/11/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies registry modification of the LocalAccountTokenFilterPolicy value. A compromised local administrator account
+can provide means for an attacker to move laterally between domain systems. With User Account Control enabled, filtering
+the privileged token for local administrator accounts will prevent the elevated privileges of these accounts from being
+used over the network.
```

Comment:
I think we can reword the second part to describe how it works first and why an attacker may want to disable it after

Reply:
reworded to be more specific.